tests/runtime_neutral/test_check_installed_runtime_root.py

```
@@ -26,50 +27,87 @@ def resolve_powershell() -> str | None:
     return None
 
 
+def _to_bash_path(path: Path) -> str:
+    resolved = str(path.resolve()).replace("\\", "/")
+    if len(resolved) >= 3 and resolved[1:3] == ':/':
+        return f"/mnt/{resolved[0].lower()}/{resolved[3:]}"
+    return resolved
+
+
+def _to_windows_path(path: Path) -> str:
+    converted = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)